be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -60,7 +60,7 @@ Status AggregateBlockingSinkOperator::push_chunk(RuntimeState* state, const vect
              _aggregator->conjunct_ctxs().empty() &&       // no 'having' clause
              _aggregator->get_aggr_phase() == AggrPhase2); // phase 2, keep it to make things safe
     const auto chunk_size = chunk->num_rows();
-    DCHECK_LE(chunk_size, config::vector_chunk_size);
+    DCHECK_LE(chunk_size, state->query_options().batch_size);
```

Comment:
why not directy use state->batch_size();

Reply:
Have been fixed